packages/graphback-runtime/src/data/RESTDataProvider.ts

```diff
@@ -0,0 +1,127 @@
+import { GraphbackDataProvider } from './GraphbackDataProvider';
+const fetch = require('node-fetch');
+
+export class RESTDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
+    // Base url of the endpoint. When creating an RESTDataProvider instance, a string is passed into the constructor.
+    protected baseUrl : string;
+
+    //constructor
+    public constructor(baseUrl:string) {
+      this.baseUrl = baseUrl;
+    }
+
+    /** 
+     * @param data - JSON object which carries the data needed for the create operation
+     * @param context - Context carries the path needed to create a valid endpoint url
+     * 
+     * valid url => (baseUrl+path) 
+     * baseUrl is defined at instance creation via the constructor. path is passed at runtime via 
+     * the context parameter.
+     * 
+     */
+    async create(data: Type, context?: GraphbackContext): Promise<Type> {
+        const url = this.baseUrl+`/${context!==undefined ? context : ""}`   
+        const res = await fetch(url, {
+            method: 'post',
+            body:    JSON.stringify(data),
+            headers: { 'Content-Type': 'application/json' },
```

Comment:
It should allow peopel to pass headers as well.

Reply:
@wtrocki So we can add headers to data object passed into the method and access headers from that. Right ?

Reply:
Constructor might work (but not always as we might know token later)